backend/inspirehep/snow/views.py

```diff
@@ -0,0 +1,121 @@
+import structlog
+from flask import Blueprint
+from webargs import fields
+from webargs.flaskparser import FlaskParser
+
+from inspirehep.accounts.decorators import login_required_with_roles
+from inspirehep.accounts.roles import Roles
+from inspirehep.serializers import jsonify
+from inspirehep.snow.api import InspireSnow
+from inspirehep.snow.errors import CreateTicketException, EditTicketException
+
+blueprint = Blueprint("inspirehep_tickets", __name__, url_prefix="/tickets")
+LOGGER = structlog.getLogger()
+parser = FlaskParser()
+
+
+@blueprint.route("create", methods=["POST"])
+@parser.use_args(
+    {
+        "functional_category": fields.String(required=True),
+        "recid": fields.String(required=False),
+        "subject": fields.String(required=True),
+        "description": fields.String(required=False),
+        "caller_email": fields.String(required=False),
+    }
+)
+@login_required_with_roles([Roles.superuser.value])
+def create_ticket(args):
+    snow_instance = InspireSnow()
+    try:
+        ticket = snow_instance.create_inspire_ticket(
+            user_email=args.get("caller_email"),
+            functional_category=args.get("functional_category"),
+            subject=args.get("subject"),
+            description=args.get("description"),
+        )
+        return jsonify({"ticket_id": ticket}), 200
+    except (CreateTicketException, EditTicketException):
+        return jsonify({"message": "Can't create SNOW ticket!"}), 500
+
+
+@blueprint.route("create-with-template", methods=["POST"])
+@parser.use_args(
+    {
+        "functional_category": fields.String(required=True),
+        "template": fields.String(required=True),
+        "subject": fields.String(required=True),
+        "recid": fields.String(required=False),
+        "caller_email": fields.String(required=False),
+        "template_context": fields.Dict(required=False),
+    }
+)
+@login_required_with_roles([Roles.superuser.value])
+def create_ticket_with_template(args):
+    snow_instance = InspireSnow()
+    try:
+        template_path = f"rt/{args['template']}.html"
+        ticket = snow_instance.create_inspire_ticket_with_template(
+            template_context=args.get("template_context"),
+            template_path=template_path,
+            user_email=args.get("caller_email"),
+            functional_category=args.get("functional_category"),
+            subject=args.get("subject"),
+        )
+        return jsonify({"ticket_id": ticket}), 200
+    except (CreateTicketException, EditTicketException):
+        return jsonify({"message": "Can't create SNOW ticket!"}), 500
```

Comment:
can we log what's going on in these cases?

Reply:
we can log if it's create ticket exception or edit ticket exception, I don't know if it would be very useful

Reply:
It would be helpful to understand if something is wrong on our side or snow